leftwm-core/src/utils/command_pipe.rs

```diff
@@ -90,35 +90,38 @@ async fn read_from_pipe(pipe_file: &Path, tx: &mpsc::UnboundedSender<Command>) -
 fn parse_command(s: &str) -> Result<Command, Box<dyn std::error::Error>> {
     let head = *s.split(' ').collect::<Vec<&str>>().get(0).unwrap_or(&"");
     match head {
+        "CloseWindow" => Ok(Command::CloseWindow),
```

Comment:
I like the reordering you did!
What do you think adout separating the categories with comment headlines?
Suggestions: ̀ General`, `Focus`, `Move`, `Layout`, `Workspace/Tag`
Sort stictly alphabetical inside category.

We could keep this for a separate PR though...